MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/location/LocationCameraController.java

```diff
@@ -114,6 +114,7 @@ void setCameraMode(@CameraMode.Mode final int cameraMode, @Nullable Location las
     }
 
     final boolean wasTracking = isLocationTracking();
+    mapboxMap.setUserAnimationInProgress(wasTracking);
```

Comment:
The passed flag shouldn't be based on `wasTracking`, it should be checked after the mode is set. More like this:
```
final boolean wasTracking = isLocationTracking();
this.cameraMode = cameraMode;
mapboxMap.setUserAnimationInProgress(isLocationTracking());
```

Reply:
Create a new PR to fix this https://github.com/mapbox/mapbox-gl-native-android/pull/504